en: update security config requirement in source configuration file (#667)
pingcap/docs-dm: en/source-configuration-file.md

| `port` | Specifies the port of the upstream database. |
| `user` | Specifies the username of the upstream database. |
| `password` | Specifies the user password of the upstream database. It is recommended to use the password encrypted with dmctl. |

> | `security` | Specifies the TLS config of the upstream database. The configuration file path of the certificates need to be accessed by all nodes. If the file path is local, then all the nodes in the cluster need to copy the certificates in the same path of each host.|

Reviewer: ```suggestion
| `security` | Specifies the TLS config of the upstream database. The configured file paths of the certificates must be accessible to all nodes. If the configured file paths are local paths, then all the nodes in the cluster need to store a copy of the certificates in the same path of each host.|
```
